packages/angular/src/lib/stencil-generated/components.ts

```diff
@@ -1037,9 +1087,13 @@ export declare interface GcdsTextarea extends Components.GcdsTextarea {
    */
   gcdsBlur: EventEmitter<CustomEvent<void>>;
   /**
-   * Update value based on user input.
+   * Emitted when the textarea has received input.
```

Comment:
Is this meant for `gcdsInput`?
```suggestion
   * Emitted when the textarea value has changed.
```